### ISSUE TITLE
Black screen after Booting Log show something call 'EFI vga' on  Dell Inspiron 3162/3164

### ISSUE THREAD

**Issue by klogod** [sebanc/brunch#669]
When I select to Boot Chrome OS from the grub menu, everything is loading fine, but the screen just go black after something call 'XXXXXEFI vga'. 
It's too fast so i cant read the whole log.
Grub menu is normal.

I installed the r86 version with rammus recovery.

**Comment by sebanc** [comment]
You should have received a github email invite to "brunch-testing" repository, could you try the build with 5.4 kernel there ?

**Comment by klogod** [comment]
> You should have received a github email invite to "brunch-testing" repository, could you try the build with 5.4 kernel there ?

OK, I will try.

**Comment by mirek190** [comment]
Hello sebanc .
Can you add me to brunch-testing as well ? 
I also want to make tests with new kernel 5.x line .

**Comment by sebanc** [comment]
@mirek190 Github email invite sent.

**Comment by mirek190** [comment]
Thanks sebanc

**Comment by klogod** [comment]
Its's work now, thanks sebanc

**Comment by xp1ode** [comment]
anyway to get an invite to the brunch-testing as well?

**Comment by sebanc** [comment]
@xp1ode Sure, Github email invite sent.

**Comment by Bigwil63** [comment]
Having the same issue on Microsoft Surface 3.  Everything great and then a normal reboot caused the "...EFI vga" in the boot log.  It does boot, but is painfully slow and unusable (never made it past the login).  Tried to boot from the original USB and same thing.  Maybe the 5.4 kernel would help me too.

**Comment by klogod** [comment]
Try to disable all about "legacy ROM" in BIOS.

> Having the same issue on Microsoft Surface 3. Everything great and then a normal reboot caused the "...EFI vga" in the boot log. It does boot, but is painfully slow and unusable (never made it past the login). Tried to boot from the original USB and same thing. Maybe the 5.4 kernel would help me too.

**Comment by Bigwil63** [comment]
I don't have that option on the surface 3, I am familiar with it on other PCs.  TPM and Secure Boot Control are both disabled though.  Experimented with turning them on and back off with no change.  Thanks

**Comment by sebanc** [comment]
@Bigwil63 you should have received a brunch-testing github email invite.

**Comment by Pentahex** [comment]
I would also request a brunch-testing invite, love this project, I have it installed on three of my laptops.

**Comment by stateandrei5** [comment]
Can i get one too?

**Comment by sebanc** [comment]
@stateandrei5 the newest release is now in the main brunch repo

**Comment by stateandrei5** [comment]
> @stateandrei5 the newest release is now in the main brunch repo

yep. im clean installing. currently out for a smoke while waiting on framework rebuild
